lib/rollout.rb

```diff
@@ -159,6 +166,33 @@ def define_group(group, &block)
     @groups[group.to_sym] = block
   end
 
+  def define_id_collection(name, ids = [])
+    @storage.set("collection:#{name}", ids.join(","))
+    @storage.set(collection_key, (collections.keys | [name.to_sym]).join(","))
+  end
+
+  def add_collection_to_feature(collection, feature)
+    with_feature(feature) do |feature|
+      @storage.set(collection_feature_key(feature.name), (feature_collections(feature.name).keys | [ collection.to_sym]).join(","))
+    end
+  end
+
+  def collections
+    Hash[*retrieve_from_storage(collection_key).map{ |collect| [ collect.to_sym, @storage.get("collection:#{collect}") ] }.flatten]
```

Comment:
oh, now it makes sense. It was just that i was expecting "101,102" to be an array, not a string. what do you think about this approach? We tend to use ```each_with_object``` instead of this ```Hash[*ary]```, cause it's easier to understand in most cases. Maybe something like:

```ruby
retrieve_from_storage(collection_key).each_with_object({}) do |collection_name, obj|
  obj[collection_name.to_sym] = @storage.get(collection_key(collection_name))
end
```
I think these way you can really see how the hash is being constructed.